intake/utils.py

```diff
@@ -112,7 +112,7 @@ def __getstate__(self):
         # OrderedDict steers dask toward a different and faster tokenization.
         kwargs = collections.OrderedDict({k: arg.__getstate__()
                   if isinstance(arg, DictSerialiseMixin) else arg
-                  for k, arg in self._captured_init_kwargs.items()})
+                  for k, arg in sorted(self._captured_init_kwargs.items(), key=lambda x: x[0])})
```

Comment:
Keyword argument order has meaning, so these should not be sorted. Two objects with `_captured_init_kwargs` given in a different order should tokenize uniquely. This was introduced here, for reference: https://github.com/intake/intake/pull/455#issuecomment-574143114